runtime/onert/core/include/ir/Model.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __ONERT_IR_MODEL_H__
+#define __ONERT_IR_MODEL_H__
+
+#include "ir/ModelOperandIndexSequence.h"
+#include "ir/Subgraphs.h"
+
+namespace onert
+{
+namespace ir
+{
+
+struct Model
+{
+  Model(const std::shared_ptr<Subgraphs> &subgs) : _subgraphs(subgs) {}
+
+  std::shared_ptr<Subgraphs> _subgraphs;
+  ModelOperandIndexSequence _inputs;
```

Comment:
Here, what index is represented by ModelOperandIndexSequence ?

Reply:
@chunseoklee I thought Model is assumed to have its inputs and outputs only from primary subgraph (= subgraph 0) like ordinary `.tflite` and `.circle`. Today, I also got confirmed that @hseok-oh also had same assumption from #6274.